wallet/wallet.c

```diff
@@ -1930,7 +1952,18 @@ void wallet_channel_save(struct wallet *w, struct channel *chan)
 	}
 	db_bind_amount_msat(stmt, 42, &chan->htlc_minimum_msat);
 	db_bind_amount_msat(stmt, 43, &chan->htlc_maximum_msat);
-	db_bind_u64(stmt, 44, chan->dbid);
+
+	if (chan->alias[LOCAL] != NULL)
+		db_bind_short_channel_id(stmt, 44, chan->alias[LOCAL]);
+	else
+		db_bind_null(stmt, 44);
+
+	if (chan->alias[REMOTE] != NULL)
+		db_bind_short_channel_id(stmt, 45, chan->alias[REMOTE]);
```

Comment:
Hmm, note that db_bind_short_channel_id should be renamed db_bind_short_channel_id_str and similarly deprecated...